.ci/jenkins/build-test/Jenkinsfile

```diff
@@ -21,6 +21,71 @@ pipeline {
                      required: true)
     }
     stages {
+        stage('Scan for possible secrets'){
+            steps{
+                sh label: "Secrets scan",
+                   script: '''
+                   #!/bin/bash
+
+                   set +x
+
+                    if [ ! -f .secrets.baseline ] ;
+                    then
+                        # This generated baseline file will only be temporarily available on the GitHub side and will not appear in the user's local files.
+                        # Scanning an empty folder to generate an initial .secrets.baseline without secrets in the results.
+                        echo "⚠️ No existing .secrets.baseline file detected. Creating a new blank baseline file."
+                        mkdir empty-dir
+                        detect-secrets scan empty-dir > .secrets.baseline
+                        echo "✅ Blank .secrets.baseline file created successfully."
+                        rm -r empty-dir
+                    else
+                        echo "✅ Existing .secrets.baseline file detected. No new baseline file will be created."
+                    fi
+                    # scripts scan repository for new secrets
+                    # backup list of known secrets
+                    cp -pr .secrets.baseline .secrets.new
+                    # find secrets in the repository
+                    detect-secrets scan --disable-plugin AbsolutePathDetectorExperimental --all-files --baseline .secrets.new --exclude-files '\\.secrets..*' --exclude-files '\\.git.*' --exclude-files 'test_results'
```

Comment:
A few more candidates for what should get excluded:
```
--exclude-files '\.pytest_cache' \
--exclude-files '\.venv' \
--exclude-files 'venv' \
--exclude-files 'dist' \
--exclude-files 'build' \
--exclude-files '.*\.egg-info'
```

Reply:
Patterns have been added to the new script